src/openrct2/windows/EditorObjectSelection.cpp

```diff
@@ -588,12 +588,12 @@ static void setup_in_use_selection_flags()
 
         uint8 entryType, entryIndex;
         if (find_object_in_entry_group(&item->ObjectEntry, &entryType, &entryIndex)) {
-            if (gEditorSelectedObjects[entryType][entryIndex] & (1 << 0)) {
+            if (Editor::SelectedObjects[entryType][entryIndex] & (1 << 0)) {
                 *selectionFlags |=
                     OBJECT_SELECTION_FLAG_IN_USE |
                     OBJECT_SELECTION_FLAG_SELECTED;
             }
-            if (gEditorSelectedObjects[entryType][entryIndex] & (1 << 1)) {
+            if (Editor::SelectedObjects[entryType][entryIndex] & (1 << 1)) {
```

Comment:
Whats the meaning of (1 << 1) here? All the others use (1 << 0) or 1, might be the time to introduce some proper meaning for those.

Reply:
I created a separate issue for that.